### PR TITLE
Create akul-singhal.md

### PR DESCRIPTION
**Pull request by arcane810**

### CODE REVIEW

**Comment by agrawal-d** [comment]
When did you learn Dart?

File: _members/akul-singhal.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+layout: members
+title: Akul Singhal
+---
+
+I am a 2018 batch undergrad, pursuing Computer Science. I love taking part in competitive coding competitions and I am also interested in software development.
+
+Skills
```
Comment (agrawal-d) [inline _members/akul-singhal.md:8]: Change the `Skills ____` to `## Skills` and likewise for other subheadings.

Comment (arcane810) [reply _members/akul-singhal.md:8]: Practice School-1

**Comment by agrawal-d** [comment]
LGTM. In the future, please amend the commits instead of creating an additional commit.

I'll sqaush them this time.

Thanks for working on this!

**Pull request merged**